File: backend/Models/ProfileModel.js
const mongoose = require('mongoose');

const profileSchema = new mongoose.Schema({
  github: {
    type: String,
    required: true
  },
  imageSrc: {
    type: String,
    required: true
  },
  isA: {
    type: String,
    required: true
  },
  linkedIn: {
    type: String,
    required: true
  },
  location: {
    type: String,
    required: true
  },
  name: {
    type: String,
    required: false
  },
  talk1: {
    type: String,
    required: false
  },
  talk1Link: {
    type: String,
    required: false
  },
  talk2: {
    type: String,
    required: false
  },
  talk2Link: {
    type: String,
    required: false
  },
  techStack: {
    type: String,
    required: false
  },
  twtr: {
    type: String,
    required: true
  },
  woman: {
    type: String,
    required: true
  }
});

const Profile = mongoose.model('Participant', profileSchema);

module.exports = Profile;

File: backend2/models/profileModel.go
package models

import "go.mongodb.org/mongo-driver/bson/primitive"

type Profile struct {
	ID primitive.ObjectID `json:"_id,omitempty" bson:"_id,omitempty"`
	Github   string        `json:"github"`
	ImageSrc string        `json:"imageSrc"`
	IsA    string        `json:"isA"`
	LinkedIn   string        `json:"linkedIn"`
	Location string        `json:"location"`
	Name    string        `json:"name"`
	Talk1   string        `json:"talk1"`
	Talk1Link string        `json:"talk1Link"`
	Talk2    string        `json:"talk2" `
	Talk2Link   string        `json:"talk2Link" `
	TechStack string        `json:"techStack" `
	Twtr    string        `json:"twtr" `
	Woman    string        `json:"woman"`
	Experience string   `json:"experience"`
}